src/AppInstallerCLIE2ETests/Interop/InstallInterop.cs

```diff
@@ -599,5 +599,28 @@ public void GetApplicableInstaller()
             Assert.AreEqual(PackageInstallerScope.User, packageInstallerInfo.Scope);
             Assert.AreEqual("en-US", packageInstallerInfo.Locale);
         }
+
+        /// <summary>
+        /// Test install with BypassStorePolicy Enabled.
+        /// </summary>
+        /// <returns>A <see cref="Task"/> representing the asynchronous unit test.</returns>
+        [Test]
+        public async Task InstallMsixStoreBypass()

```

Comment:
This test probably will not work. We don't have any e2e tests targeting msstore types